packages/devextreme/js/localization/messages/ar.json

```diff
@@ -245,8 +245,25 @@
         "dxScheduler-editorLabelDescription": "الوصف",
         "dxScheduler-editorLabelRecurrence": "تكرار",
 
-        "dxScheduler-navigationPrevious": "Previous {0}",
-        "dxScheduler-navigationNext": "Next {0}",
+        "dxScheduler-navigationPreviousday": "Previous day",
```

Comment:
Please add separator, without it it is hard to read the name
Like `dxScheduler-navigationPrevious-day`